recipes/weeder/post-link.sh

```diff
@@ -0,0 +1,19 @@
+#!/bin/bash
+
+echo "Downloading and installing Weeder frequency files..."
+
+# Directory to store FreqFiles
+TGT="$PREFIX/share/weeder2"
+
+# Cargo port url
+CARGO_URL=http://depot.galaxyproject.org/software/
+
+# Download and unpack
+TARBALL=${PKG_NAME}_${PKG_VERSION}_src_all.tar.gz
+curl -L ${CARGO_URL}/${PKG_NAME}/$TARBALL -o $TGT/$TARBALL
```

Comment:
Yes, see [here](https://conda.io/docs/building/build-scripts.html). There's a small set of env variables available: `PREFIX`,  `PKG_NAME`, `PKG_VERSION` and `PKG_BUILDNUM`.